service/gcs/main.go

```diff
@@ -156,6 +199,21 @@ func main() {
 		logrus.WithError(err).Fatal("opengcs::main - failed add gcs pid to gcs cgroup")
 	}
 
+	gefd, err := cgrouputils.RegisterMemoryThreshold(filepath.Join(cgrouputils.MemRoot, "gcs"), gcsLimit)
+	if err != nil {
+		logrus.WithError(err).Fatal("opengcs::main - failed to register memory threshold for gcs cgroup")
```

Comment:
Let's remove the `opengcs::main - ` prefix from these log statements.

Reply:
Want me to rid all of main of these or just this commits introduced ones?

Reply:
I'm fine with fixing things as you come across them. If you're doing something more broadly across the repo it would be better in a separate PR.

Reply:
I'll just add a separate commit to remove them in this pr as they're all in main, might as well I feel.